### PR TITLE
add cvxportfolio

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by timkpaine** [comment]
@conda-forge/help-python bit of an ugly wheel-only one but ready for a look

File: recipes/cvxportfolio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{% set version = "0.0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # no sdist provided, no tags on github
+  url: https://files.pythonhosted.org/packages/e5/f3/4058567aa62a04663fcd195f37d407c6dddd52fca093341ba49b3db21031/cvxportfolio-0.0.12-py3-none-any.whl
+  sha256: 4079964dcc755612a9688232710e9dd94c6bcb9929b3bc56098bc0323fbcdb5d
```
[the remark this answers is not in the document]
Comment (timkpaine) [reply recipes/cvxportfolio/meta.yaml:11]: I have already started a maintenance PR but this should still go as-is as it is the canonical source for the version. Even though it's a wheel it's not compiled, it's just vanilla python.

File: recipes/cvxportfolio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% set name = "cvxportfolio" %}
+{% set version = "0.0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # no sdist provided, no tags on github
+  url: https://files.pythonhosted.org/packages/e5/f3/4058567aa62a04663fcd195f37d407c6dddd52fca093341ba49b3db21031/cvxportfolio-0.0.12-py3-none-any.whl
```
Comment (ocefpaf) [inline recipes/cvxportfolio/meta.yaml:10]:
```suggestion
  url: https://pypi.io/packages/py3/c/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
```

File: recipes/cvxportfolio/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{% set name = "cvxportfolio" %}
+{% set version = "0.0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # no sdist provided, no tags on github
```
Comment (ocefpaf) [inline recipes/cvxportfolio/meta.yaml:9]: Please add the URL for the issue upastream.

File: recipes/cvxportfolio/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - cvxpy >=1.0.6
+    - matplotlib-base
+    - numpy
+    - pandas
+    # Missing from upstream
```
Comment (ocefpaf) [inline recipes/cvxportfolio/meta.yaml:28]: If you opened an issue for this please add the link here.

File: recipes/cvxportfolio/meta.yaml (hunk 3 above)
Comment (ocefpaf) [reply recipes/cvxportfolio/meta.yaml:9]: Can you address this comment?

Comment (timkpaine) [reply recipes/cvxportfolio/meta.yaml:9]: It was already added before your last changes

File: recipes/cvxportfolio/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  # no sdist provided, no tags on github
+  # see https://github.com/cvxgrp/cvxportfolio/issues/67
+  fn: wheel.whl
+  url: https://pypi.io/packages/py3/c/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
+  sha256: 4079964dcc755612a9688232710e9dd94c6bcb9929b3bc56098bc0323fbcdb5d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install wheel.whl -vv --no-deps
```
[the remark this answers is not in the document]
Comment (timkpaine) [reply recipes/cvxportfolio/meta.yaml:18]: Ok no worries, will revert later today back to what it did before once I get to my computer

**Comment by timkpaine** [comment]
thanks @ocefpaf!

**Pull request merged**